src/player.cpp

```diff
@@ -10551,6 +10550,21 @@ bool player::is_wearing_shoes(std::string side) const
     return (left && right);
 }
 
+bool player::is_wearing_helmet() const
+{
+    bool ret = false;
+    for (auto &i : worn) {
```

Comment:
This should be properly formatted per current astyle rules used on the project (even if the file you working with is not yet properly formatted).